### ISSUE TITLE
Cosmetic armor crash

### ISSUE THREAD

**Issue by tlitookilakin** [benblank/Ice#8]
When I died, my game crashed with a `NoClassDefFoundError: lain/mods/cos/PlayerUtils` error. It may just be a problem with specific versions of cosmetic armor. full log here: http://pastebin.com/YaXSCXrt I'm running the newest curse-available versions of both. (1.1.5 and v2c, accordingly)
I'll backdate cosmetic armor and let you know if the bug goes away.

**Comment by tlitookilakin** [comment]
Yep. I think the newest version of Cosmetic Armor has a different class structure. Compatibility works fine with v1b installed.

**Comment by benblank** [comment]
I bet you're right.  Either way, though, crashing is _not_ the correct response!  I'll get a new version out ASAP.
